src/components/Header/Header.module.scss

```diff
@@ -1,15 +1,19 @@
 .header {
+    margin-top: 0.5rem;
     display: flex;
     justify-content: center;
-    align-items: center;
     flex-wrap: wrap;
-    padding-top: 0.625rem;
+    padding: 1rem;
+    align-items: center;
+    gap: 1rem;
+    border-bottom: 1px solid var(--toastify-text-color-light);
 }
 
 .link {
-    padding: 0.625rem;
+    padding: 0.5rem;
+    width: 6rem;
     text-decoration: none;
-    font-weight: bold;
+    font-size: 14px;
```

Comment:
Can you use REM instead of PX if the value is less than 8px?

Reply:
1rem = 16px, So if I'm not wrong we can use rem when greater than or equal to 16px. But in this case 14px = 0.875 rem.

Reply:
Could you share resources for the above statement please